Packs/SplunkPy/Integrations/SplunkPy/SplunkPy.py

```diff
@@ -163,7 +164,7 @@ def updateNotableEvents(sessionKey, baseurl, comment, status=None, urgency=None,
     args['output_mode'] = 'json'
 
     mod_notables = requests.post(baseurl + 'services/notable_update', data=args, headers=auth_header,
-                                 verify=VERIFY_CERTIFICATE)
+                                 verify=VERIFY_CERTIFICATE, proxies=PROXIES)
```

Comment:
This is not needed. Calling `handle_proxy()` at the start is the only thing that is needed when working with requests.